Automattic/simplenote-ios
# SearchResultsViewController: KeyboardObservable

### Details:
In this PR we're making the new **SearchResultsViewController** react to Keyboard events, and apply.

Ref. #507 

@SergioEstevao May I bug you with a super quick PR?
Thanks in advance!!

### Test
0. Set the `FeatureManager.advancedSearchEnabled` property to `true`
1. Log into your account
2. Press over the **Search Bar** to get the SearchResultsViewController onscreen

- [ ] Verify that the bottom row in your search results is visible. The keyboardWillShow event is expected to apply a bottom padding!

### Release
These changes do not require release notes.
